packages/webcomponents/src/components/table/table.tsx

```diff
@@ -80,6 +86,7 @@ export class Table {
                         this.rowData?.map((data, index) => {
                           return (
                             <tr
+                              data-test-id="payment-row"
```

Comment:
Should we pass test ids through as props? Are there others?

Reply:
Good one, I renamed it to `table-row`